lib/round.rb

```diff
@@ -0,0 +1,79 @@
+class Round
+  attr_reader :deck, :turns, :deck_index
```

Comment:
you never need to call `.deck_index` on an instance of the round class in other classes, so `:deck_index` does not need to be an  attr_reader.

Reply:
it's kind of like how you don't have an attr_Reader for number_correct -- you just need that value within the class, but nowhere else.